src/arrays/Arrays.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace arrays;
+
+class Arrays implements ArraysInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function repeatArrayValues(array $input): array
+    {
+        $new_array = [];
+        if (count($input) != 0) {
+            foreach ($input as $currentElement) {
```

Comment:
please use php array function

Reply:
yes, it is just for learning about php array functions and how to use them.
Main point it still should work and tests should pass.

Reply:
> yes, it is just for learning about php array functions and how to use them. Main point it still should work and tests should pass.

All changes done. Now I'm proud of my code here ;)